velox/type/tests/SubfieldTest.cpp

```diff
@@ -154,3 +148,52 @@ TEST(SubfieldTest, longSubscript) {
   ASSERT_TRUE(longSubscript);
   ASSERT_EQ(longSubscript->index(), 3309189884973035076);
 }
+
+class FakeTokenizer : public Tokenizer {
+ public:
+  explicit FakeTokenizer(const std::string& path) : path_(path) {
+    state = State::kNotReady;
+  }
+
+  bool hasNext() override {
+    switch (state) {
+      case State::kDone:
+        return false;
+      case State::kReady:
```

Comment:
Seems only kDone and kNotReady are needed.

Reply:
Updated, thanks.